src/lazy.jl

```diff
@@ -627,14 +627,14 @@ isbigfloat(x::NumberResult) = x.tag == BIGFLOAT
             # if we overflowed, then let's try BigFloat
             bres = Parsers.xparse2(BigFloat, buf, startpos, len)
             if !Parsers.invalid(bres.code)
-                return NumberResult(bres.val), startpos + bres.tlen
+                return NumberResult(bres.val), Int(startpos + bres.tlen)
```

Comment:
```suggestion
                return NumberResult(bres.val), startpos + Int(bres.tlen)
```

Ah, this is the problem. `bees.tlen` is an `::Int64`:

https://github.com/JuliaData/Parsers.jl/blob/6352c4bbce6f9df8cea268a39a3c43e1ee4944e7/src/Parsers.jl#L31